src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,39 @@
 package core.basesyntax;
 
 public class MainApp {
+    private final Machine truck = new Truck();
 
+    private final Machine excavator = new Excavator();
+
+    private final Machine bulldozer = new Bulldozer();
```

Comment:
The fields `truck`, `excavator`, `bulldozer` should have access modifiers that limit their visibility as per encapsulation best practices. Consider making them private if they are not intended to be accessed from outside the class.